CRM/Civicase/Service/CaseMenu/MainRoute.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Class for Main route.
+ *
+ * Represents the main item on the menu, containing all sub items.
+ */
+class CRM_Civicase_Service_CaseMenu_MainRoute extends CRM_Civicase_Service_CaseMenu_Route {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function label() {
+    return ucfirst(strtolower($this->categoryName));
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function name() {
+    return $this->categoryName;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function url() {
+    return NULL;
```

Comment:
As its url, should not we return empty string?

Reply:
In my opinion, NULL seems correct. Anyway, this is later used for creating a navigation URL, which works with NULL (based on the previous code)